FirebaseAuth/Sources/FirebaseAuth.private.modulemap

```diff
@@ -0,0 +1,4 @@
+framework module FirebaseAuth_Private {
```

Comment:
Is the file needed in git? Versus generating it completely?

Reply:
No we could generate it completely.

Reply:
File has since been removed.